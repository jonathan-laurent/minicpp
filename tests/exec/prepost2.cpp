#include <iostream>

int main() {
  int x = 41;
  std::cout << "x = " << x << "\n";
  std::cout << "x = " << x++ << "\n";
  std::cout << "x = " << ++x << "\n";
  std::cout << "x = " << x-- << "\n";
  std::cout << "x = " << --x << "\n";
  std::cout << "x = " << x << "\n";
}
