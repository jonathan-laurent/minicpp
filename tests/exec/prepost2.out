x = 41
x = 41
x = 43
x = 43
x = 41
x = 41
